src/components/commons/inputs/ImageField.module.scss

```
@@ -0,0 +1,59 @@
+.image-field {
+  @include column-flexbox($gap: 1.6rem);
+
+  width: 100%;
+
+  &-group {
+    @include column-flexbox;
+
+    width: 100%;
+    height: 12rem;
+
+    background: $input-background;
+    border: 0.1rem dashed $input-stroke;
+    border-radius: 0.8rem;
+
+    &-title {
+      @include text-style(14, $gray40);
+
+      &.active {
+        color: $purple;
+      }
+    }
+  }
+
+  &-name-list {
+    @include column-flexbox(center, start, 0.8rem);
+
+    width: 100%;
+    margin-right: auto;
+
+    &-item {
+      @include flexbox(between);
+
+      width: 100%;
+      padding-bottom: 0.8rem;
+      border-bottom: 0.1rem solid $black60;
+
+      .item-group {
+        @include flexbox($gap: 0.8rem);
+
+        .image-group {
+          @include flexbox($gap: 0.2rem);
+
+          .file-name {
+            @include text-style(12, $gray10);
+          }
+        }
```

```suggestion
        .image-group {
          @include flexbox($gap: 0.2rem);
          @include text-style(12, $gray10);
        }
```

IMHO,
여기 text-style 은 위의 image-group 에다가 적어놔도 file-name에 적용되지 않을까요?
그러면 중첩구문의 깊이가 조금은 줄어들거 같습니다!

---

좋은 생각인 것 같습니다 수정해보겠습니다~